content/en/references/configuration.md

```diff
@@ -226,6 +226,7 @@ Please consult the [migration guide]({{< ref "user-guide/aws/lambda#migrating-to
 | `LAMBDA_RUNTIME_IMAGE_MAPPING` | [base images for Lambda](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-images.html) (default) | Customize the Docker image of Lambda runtimes, either by:<br> a) pattern with `<runtime>` placeholder, e.g. `custom-repo/lambda-<runtime>:2022` <br> b) json dict mapping the `<runtime>` to an image, e.g. `{"python3.9": "custom-repo/lambda-py:thon3.9"}` |
 | `LAMBDA_SYNCHRONOUS_CREATE` | `0` (default) | Set to `1` to create lambda functions synchronously (not recommended). |
 | `LAMBDA_TRUNCATE_STDOUT` | `2000` (default) | Allows increasing the default char limit for truncation of lambda log lines when printed in the console. This does not affect the logs processing in CloudWatch. |
+| `LAMBDA_RUNTIME_VALIDATION` | `0` (default) | Enable validation of the runtime environment before starting the Lambda function. Set to `1` to ensure you are not using an unsupported/deprecated runtime. |
```

Comment:
Not strictly correct. I suggest the following clarified version:

> Set to `1` to enforce strict [AWS parity](https://blog.localstack.cloud/2022-08-04-parity-explained/) by raising an exception when using a deprecated [Lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) for the API operation [CreateFunction](https://docs.aws.amazon.com/lambda/latest/api/API_CreateFunction.html). Deprecated Lambda runtimes (e.g., `nodejs14.x`) can be used with disabled validation (current default).


Notice that full parity for [UpdateFunctionConfiguration](https://docs.aws.amazon.com/lambda/latest/api/API_UpdateFunctionConfiguration.html) mirroring the staged deprecation phase at AWS (i.e., block create, block update, etc) is not supported. It's a rabbit hole with diminishing benefits for LocalStack.


Full details are in the backlog item "Add Lambda option to disable runtime validations":
- `LAMBDA_RUNTIME_VALIDATION=1` enforces strict AWS parity by raising an exception. In the LocalStack logs, it prints a hint about the configuration to disable validation for using deprecated runtimes.
- `LAMBDA_RUNTIME_VALIDATION=0` does not validate the string value of the provided Lambda runtime and enables creating functions for deprecated runtimes. It does still raises an exception if no implementation is available.